src/roompanel/RoomManager.cpp

```diff
@@ -0,0 +1,252 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (C) 2021 The MMapper Authors
+// Author: Massimiliano Ghilardi <massimiliano.ghilardi@gmail.com> (Cosmos)
+
+#include <memory>
+#include <string>
+#include <utility>
+#include <QtCore>
+
+#include "../global/StringView.h"
+#include "../proxy/GmcpMessage.h"
+#include "RoomManager.h"
+#include "RoomMobs.h"
+
+RoomManager::RoomManager(QObject *const /*parent*/)
+    : QObject{nullptr}
+    , m_room{std::make_unique<RoomMobs>(nullptr)}
```

Comment:
Why not make it here? The parent can be the RoomManager ?

Reply:
Fixed. I set RoomMobs's parent to `parent` i.e. the main window